src/app/tests/suites/certification/Test_TC_TCCM_3_1.yaml

```diff
@@ -21,183 +20,111 @@ PICS:
-          [1692182446.526642][3967:3969] CHIP:TOO:       ModeTags: 1 entries
-          [1692182446.526670][3967:3969] CHIP:TOO:         [1]: {
-          [1692182446.526695][3967:3969] CHIP:TOO:           Value: 16384
-          [1692182446.526717][3967:3969] CHIP:TOO:          }
-          [1692182446.526741][3967:3969] CHIP:TOO:      }
-          [1692182446.526773][3967:3969] CHIP:TOO:     [3]: {
-          [1692182446.526795][3967:3969] CHIP:TOO:       Label: Rapid Freeze
-          [1692182446.526817][3967:3969] CHIP:TOO:       Mode: 2
-          [1692182446.526842][3967:3969] CHIP:TOO:       ModeTags: 1 entries
-          [1692182446.526870][3967:3969] CHIP:TOO:         [1]: {
-          [1692182446.526894][3967:3969] CHIP:TOO:           Value: 16385
-          [1692182446.526917][3967:3969] CHIP:TOO:          }
-          [1692182446.526940][3967:3969] CHIP:TOO:      }
-
-
-          Set the OnMode attribute value to OnMode_value
-
-          ./chip-tool refrigeratorandtemperaturecontrolledcabinetmode write on-mode 0 1 1
-
-          On TH(chip-tool) log, Verify DUT responds with a SUCCESS (value 0x00) status response and below is the sample log provided for the raspi platform:
-
-          [1692170913.264454][3071:3073] CHIP:DMG:                         StatusIB =
-          [1692170913.264489][3071:3073] CHIP:DMG:                         {
-          [1692170913.264522][3071:3073] CHIP:DMG:                                 status = 0x00 (SUCCESS),
-          [1692170913.264557][3071:3073] CHIP:DMG:                         },
-      disabled: true
+    - label: "Wait for the commissioned device to be retrieved"
+      cluster: "DelayCommands"
+      command: "WaitForCommissionee"
+      arguments:
+          values:
+              - name: "nodeId"
+                value: nodeId
 
     - label:
-          "Step 1: Commission DUT to TH (can be skipped if done in a preceding
-          test)."
-      verification: |
-
-      disabled: true
-
-    - label: "Step 2: TH reads from the DUT the OnMode attribute."
+          "Preconditions: The OnMode attribute is set to a non-NULL value from
+          the mode values indicated by the SupportedModes attribute."
       PICS: TCCM.S.A0003 && TCCM.S.F00
-      verification: |
-          ./chip-tool refrigeratorandtemperaturecontrolledcabinetmode read on-mode 1 1
+      command: "writeAttribute"
+      attribute: "OnMode"
+      arguments:
+          value: nonNull_OnMode_value
 
-          On TH(chip-tool), Verify that OnMode attribute value is an integer
-          Save the value as on_mode_dut and below is the sample log provided for the raspi platform, here OnMode value is 0
-
-          [1690190167.005388][16387:16389] CHIP:TOO: Endpoint: 1 Cluster: 0x0000_0052 Attribute 0x0000_0003 DataVersion: 473867556
-          [1690190167.005447][16387:16389] CHIP:TOO:   OnMode: 0
-      disabled: true
+    - label: "TH reads the OnMode attribute from the DUT"
```

Comment:
It  would be very helpful if the labels included the step from the test plan that this is implementing, throughout this file.

Reply:
Updated.